tests/components/advantage_air/test_climate.py

```diff
@@ -170,21 +168,6 @@ async def test_climate_async_setup_entry(hass, aioclient_mock):
     assert aioclient_mock.mock_calls[-1][0] == "GET"
     assert aioclient_mock.mock_calls[-1][1].path == "/getSystemData"
 
-    # Test set_myair service
-    await hass.services.async_call(
-        ADVANTAGE_AIR_DOMAIN,
-        ADVANTAGE_AIR_SERVICE_SET_MYZONE,
-        {ATTR_ENTITY_ID: [entity_id]},
-        blocking=True,
-    )
-    assert len(aioclient_mock.mock_calls) == 17
-    assert aioclient_mock.mock_calls[-2][0] == "GET"
-    assert aioclient_mock.mock_calls[-2][1].path == "/setAircon"
-    data = loads(aioclient_mock.mock_calls[-2][1].query["json"])
-    assert data["ac1"]["info"]["myZone"] == 1
-    assert aioclient_mock.mock_calls[-1][0] == "GET"
-    assert aioclient_mock.mock_calls[-1][1].path == "/getSystemData"
```

Comment:
Just a side-note, but this looks like that you are testing the backend protocol and not the library interface, and should therefore rather belong to the test suite of the backend library.